### DISCUSSION TITLE
How to bind class function as callback?

### DISCUSSION THREAD

**Discussion by letmp** [hsaturn/TinyMqtt#44] [Q&A] [answered]
How is it possible to bind a class function as callback without making it static?

**Comment by rimen05** [comment, marked as answer]
First of all, thank you @hsaturn for this library. It's been instrumental to introduce myself into the MQTT protocol using tiny MCUs.

That said, hi @letmp! I believe there is an example in the current release that shows a way to achieve binding of a function without making it static: examples/advanced/mqtt_class_binder/mqtt_class_binder.ino

Cheers,

**Reply by letmp** [reply]
The MqttClassBinder was introduced after I opened this thread ;)

**Reply by rimen05** [reply]
Ups. I didn't take that into consideration. Regards, :)
